### DISCUSSION TITLE
6장 프론트엔드 배포 질문...

### DISCUSSION THREAD

**Discussion by chasw326** [fsoftwareengineer/todo-application#72] [질문과 답변] [answered]
프론트엔드를 배포했는데 처음 리액트 만들때 화면이 뜹니다... 
로컬 환경에서는 잘 작동하구요...
오류가 안 떠서 어디가 잘못된지를 모르겠는데...
의심되는 부분은 빌드된 파일입니다...
생성된 build 폴더의 index.html 전문입니다..
`<!doctype html><html lang="en"><head><meta charset="utf-8"/><link rel="icon" href="/favicon.ico"/><meta name="viewport" content="width=device-width,initial-scale=1"/><meta name="theme-color" content="#000000"/><meta name="description" content="Web site created using create-react-app"/><link rel="apple-touch-icon" href="/logo192.png"/><link rel="manifest" href="/manifest.json"/><title>React App</title><script defer="defer" src="/static/js/main.68156f71.js"></script><link href="/static/css/main.073c9b0a.css" rel="stylesheet"></head><body><noscript>You need to enable JavaScript to run this app.</noscript><div id="root"></div></body></html>`
뭔가 너무 짧은거 같기도 하고...
터미널에서 명령어 실행할때 항상 todo-react-app을 기준으로 하는것이 맞나요? todo-react-app, build파일 두 곳에서 전부 eb create 해봤는데 전부 똑같이 됩니다.... 백엔드 처럼 config.yml에 deploy artifact를 적어야 하는건지... 혹시 비슷한 이슈가 있었는지 궁금합니다...
![image](https://user-images.githubusercontent.com/69625797/150743796-62db0082-85cc-4161-a066-5b676b11a9f8.png)
배포 할때 과정입니다...
config.yml
![image](https://user-images.githubusercontent.com/69625797/150753828-387640b4-8dc0-422e-868b-821974b806c0.png)

http://PROD-TODO-FRONTEND3.ap-northeast-2.elasticbeanstalk.com

**Comment by fsoftwareengineer** [comment, marked as answer]
혹시 처음 eb create하실때 git을 사용해 소스 관리를 하기로 하셨나요? 그렇다면 commit을 해야 소스가 반영됩니다.

**Reply by chasw326** [reply]
초기화 해서 다시 설정이 가능한가요?
일라스틱빈스톡 폴더를 지우고 다시 실행해봐도 이전 설정을 기억하는지 계속 똑같이 진행하네요...

**Reply by fsoftwareengineer** [reply]
git commit 
git push
를 하는 쪽이 더 빠를 것 같지만 .elasticbeanstalk 폴더를 지우고 다시 생성해도 상관없습니다. Git을 사용하지 않으신다면 .git도 지워주세요. 빈스톡이 자동으로 인식할수도 있습니다.

**Reply by hkjs96** [reply]
말씀해 주신대로 하니까 바로 되네용
